docs/querying/lookups.md

```diff
@@ -50,6 +50,10 @@ Other lookup types are available as extensions, including:
 - Globally cached lookups from local files, remote URIs, or JDBC through [lookups-cached-global](./lookups-cached-global.md).
 - Globally cached lookups from a Kafka topic through [kafka-extraction-namespace](./kafka-extraction-namespace.md).
 
+:::info
+[Multi-value dimensions](multi-value-dimensions.md) are not supported as keys in lookups. 
```

Comment:
It'll be helpful to tell people in this case, what they should do. I think the right answer is to flatten the MVD column into a list of keys and repeat the values of the lookups. This way you can construct a key-value pair list for lookups.